edx_oauth_client/backends/edx_oauth_client.py

```diff
@@ -116,51 +94,40 @@ def user_data(self, access_token, *args, **kwargs):
         if isinstance(data, list):
             data = data[0]
 
-        if data.get('success') and 'user' in data:
-            data = data['user']
-        elif 'data' in data:
-            data = data['data']
+        if data.get("success") and "user" in data:
+            data = data["user"]
+        elif "data" in data:
+            data = data["data"]
 
-        data['access_token'] = access_token
-        data.pop('password', None)
+        data["access_token"] = access_token
+        data.pop("password", None)
 
         return data
 
     def pipeline(self, pipeline, pipeline_index=0, *args, **kwargs):
-        self.strategy.session.setdefault('auth_entry', 'register')
-        return super(GenericOAuthBackend, self).pipeline(
-            pipeline=self.PIPELINE, *args, **kwargs
-        )
+        """
+        Set to session auth entry value.
+        """
+        self.strategy.session.setdefault("auth_entry", "register")
+        return super(GenericOAuthBackend, self).pipeline(pipeline=self.PIPELINE, *args, **kwargs)
```

Comment:
Method super doesn't require class reference:
```suggestion
        return super().pipeline(pipeline=self.PIPELINE, *args, **kwargs)
```